Lib/unittest/async_case.py

```diff
@@ -116,7 +116,8 @@ def _setupAsyncioRunner(self):
         assert self._asyncioRunner is None, 'asyncio runner is already initialized'
         runner = asyncio.Runner(debug=True)
         self._asyncioRunner = runner
-        asyncio.set_event_loop(runner.get_loop())
+        # Force loop to be initialized
+        runner.get_loop()
```

Comment:
`_tearDownAsyncioRunner` is called in `__del__` which seems like a lot of stuff. This could cause `set_event_loop(None)` to be called in a different thread

Reply:
It is only when you use the `debug()` method, which is not a normal way to run tests.

Reply:
Ah you mean the body of `__del__` here is only reachable when `super().debug()` crashes?

Either way calling `set_event_loop(None)` in `__del__` cannot be right. I'll raise a new ticket